tox.ini

```
@@ -1,8 +1,7 @@
 [tox]
 envlist =
-    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{5.2,6.2,6.3,6.4}
-    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.2,6.3,6.4}
-    python{3.10,3.11,3.12,3.13}-django5.1-wagtail{6.3,6.4}
+    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{5.2,6.3,6.4,7.0}
```

```suggestion
    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{6.3,6.4,7.0}
```

**Suggestion**: Remove 5.2 as EOL.